spec/factories/questions.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :question do
+    title { FFaker::HipsterIpsum.sentence.gsub(/\.$/, '?') }
+    # BUG: The author of the main repository needs to
```

Comment:
`BUG` to `BUG-1` for tracking.

Reply:
https://github.com/SpicyCookies/rails-interview-project/pull/2/commits/79e8b467992f0e8e3414c91382ded1133e6cceb2